server/routes/request.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const router = express.Router();
+const protect = require('../middleware/auth');
+const {getRequests,postRequest,updateRequest} = require('../controllers/request')
+
+router.route("/").get(protect,getRequests).post(protect,postRequest);
```

Comment:
```suggestion
router.route("/").get(protect,getRequests);
router.route("/").post(protect,postRequest);
```
No effect on functionality here, just a bit more readable